test_data/Zilien version 1/2022-04-06 16_17_23 full set_01_02_CVA_DUSB0_C01.mpt

```diff
@@ -0,0 +1,938 @@
+EC-Lab ASCII FILE

```

Comment:
Likewise, I wonder if it's worth thinning out the data here like in Pt_poly_cv_CUT.mpt?

Reply:
Maybe. The thing is, that the biologic ones are not even used for now, so maybe I should just delete them and delete them from the history. What do you think?

It's another story about the tsv, see below.

Reply:
It's not a big deal with space, as you said, and maybe someone will be interested in them for writing other tests. I think fine to leave in the repository as is.